Core/src/main/java/com/paypal/android/core/graphql/common/GraphQLClient.kt

```diff
@@ -0,0 +1,45 @@
+package com.paypal.android.core.graphql.common
+
+import com.paypal.android.core.APIClientError
+import com.paypal.android.core.CoreConfig
+import com.paypal.android.core.GraphQLRequestFactory
+import com.paypal.android.core.Http
+import org.json.JSONObject
+import java.net.HttpURLConnection
+
+internal interface GraphQLClient {
```

Comment:
Do we need this interface? `mockk` should allow us to mock any type without needing a separate `Impl` class.